doc/src/assets/cover.tex

```diff
@@ -0,0 +1,48 @@
+%% ============================================================================
+%% Custom tex styles, including this file, add a custom cover to the document.
+%%
+%% These custom styles include:
+%%  - `cover.tex`: This file, The main definition of the cover, 
+%%      used to replace the default `\maketitle` command.
+%%  - `custom.sty`: Load the macro package required for the cover, 
+%%      define the background image style, etc.
+%%  - `preamble.tex`: Replace the default preamble for inserting a custom cover.
+%%  - `logo.png`: logo of julia. (LaTeX does not yet support SVG)
+%%  - `cover-background.png`: Background image of the cover title, 
+%%      from julia's homepage.
+%% ============================================================================
+
+%% ---- reset page geometry for cover page
+\newgeometry{left=2cm,right=2cm,bottom=3cm}
+% ref: memman@v3.7q, P65, "4.1. Styling the titling"
+%   http://mirrors.ctan.org/macros/latex/contrib/memoir/memman.pdf
+\begin{titlingpage}
+    % set background image
+    \BgThispage
+    \vspace*{2cm}\noindent
+
+    %% Main Heading & Secondary Heading
+    \textcolor{white}{ \MainHeading  \DocMainTitle } 
+    \\[0.6cm]
+    \textcolor{white}{
+        \SecondaryHeading A fresh approach to technical computing.
```

Comment:
I believe this slogan was removed (in the push for 1.0 IIRC?) so should not be included here either.

Reply:
Done.

Reply:
@fredrikekre How about a different sentence? Using "A guide to the language and stdlib" as @tecosaur suggested.

Reply:
It isn't really a guide though. Why is there a need to have a sub heading?